File: 1013-fibonacci-number/fibonacci-number.php
class Solution {

    /**
     * @param Integer $n
     * @return Integer
     */
    function fib($n) {
        
       if ($n == 1 || $n == 0) {
            return $n;
        }

        $fnm1 = $this->fib($n - 1);
        $fnm2 = $this->fib($n - 2);
        $fn = $fnm1 + $fnm2;

        return $fn;

    }
}

File: 1013-fibonacci-number/fibonacci-number.java
class Solution {
    public int fib(int n) {
        // base case 
     if(n==0|| n==1){

         return n;
 
     }
       int fibnm1 = fib(n-1);
       int fibnm2 = fib(n-2);
         
         int fibn= fibnm1 + fibnm2;


       return fibn;

        
    }
}